ocd-3508.sql

```diff
@@ -0,0 +1,57 @@
+DROP TABLE IF EXISTS openchpl.certification_result_svap;
+DROP TABLE IF EXISTS openchpl.svap_criteria_map;
+DROP TABLE IF EXISTS openchpl.svap;
+
+CREATE TABLE openchpl.svap (
+    id bigserial NOT NULL,
+    regulatory_text_citation varchar(30) NOT NULL,
```

Comment:
Might want to check on max length for theses values. Not sure what it will be

Reply:
The largest in length in the spreadsheet is 13 chars.  I used 30 chars since that it what we used for the `criterion.number`.